ScnSoft-Education/LeetCodeTraining.Tests/RemoveElementTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using LeetCodeTraining.RemoveElement;
+
+namespace LeetCodeTraining.Tests
+{
+    public class RemoveElementSolutionTest
+    {
+        [Theory]
+        [InlineData(new int[] { 3, 2, 2, 3 }, 3, 2)]
+        [InlineData(new int[] { 0, 1, 2, 2, 3, 0, 4, 2 }, 2, 5)]
+        public void RemoveElement(int[] nums, int val, int expectedResult)
+        {
+            //Act
+            int result = RemoveElementSolution.RemoveElement(nums, val);
```

Comment:
Please add test on checking that first k elements are expected not equal to val.

![image](https://github.com/ArinaSiselina/ScnSoft-Education/assets/101803780/4dd54d8d-0262-4874-9b39-24853eaf12d5)